modules/learn.py

```diff
@@ -26,7 +26,7 @@ def learn():
 
 # getting words
 words = {}
-with open('resources/vocab-words.txt') as fp:
+with open('/home/rishabh/Developer/python/dude/resources/vocab-words.txt') as fp:
```

Comment:
This won't work for others' systems. Please remove this part. 
We will fix this in a different PR
For now, focus on the Issue you are fixing